resources/campaigns/Battle_for_the_UAE.json

```diff
@@ -5,7 +5,7 @@
   "recommended_player_faction": "Bluefor Modern",
   "recommended_enemy_faction": "Iran 2015",
   "description": "<p>Following the Battle of Abu Dhabi, Iran's invasion of the UAE has been halted approximately 20 miles Northeast of Liwa Airbase by coalition forces.</p><p>After weeks of stalemate, coalition forces have consolidated their position and are ready to launch their counterattack to push Iranian forces off the peninsula.</p>",
-  "version": "8.0",
+  "version": "8.1",

```

Comment:
You don't need to update this for campaigns that aren't actually using the new feature, btw. Bumping this to 8.1 will mark it incompatible with Liberation 4.1 since that'll be running campaign version 8.0, even though missions without helipads are still compatible. Since these are the bundled campaigns it doesn't make much difference, although I suppose it's possible that someone may want to copy the new version of a campaign from develop into Liberation 4.1, and this would soft-prevent (the UI would mark them incompatible and hide them by default) that.

Super minor, just an FYI.

Reply:
Resolved in ce1c416b2000126455be86b15e6115d97f496b23